internal/wrphandlers/qos/priority_queue.go

```diff
@@ -0,0 +1,60 @@
+// SPDX-FileCopyrightText: 2024 Comcast Cable Communications Management, LLC
+// SPDX-License-Identifier: Apache-2.0
+
+package qos
+
+import (
+	"github.com/xmidt-org/wrp-go/v3"
+)
+
+// Item contains a wrp Message pointer and is managed by PriorityQueue.
+type Item struct {
+	value    *wrp.Message
+	priority wrp.QOSValue
+	index    int
```

Comment:
Why is there an index?  This seems like it's not really important.

Reply:
forgot to remove this, used it for a quick test